app/mailers/user_mailer.rb

```diff
@@ -75,7 +75,7 @@ def poke_inactive(user:)
     return unless set_and_check_user(user)
     @most_repo   = Repo.order_by_issue_count.first
     @need_repo   = Repo.order_by_need.not_in(@most_repo.id).first
-    @random_repo = Repo.rand.not_in(@most_repo.id, @need_repo.id).first
+    @random_repo = Repo.rand.first
```

Comment:
@GabrielRMuller The purpose here is not send the inactive user a repo which needs most support. So we can use a fallback here instead of selecting completely randomly. We can check if a random repo which is not needing support and only if it does not exist, we can send the most_repo or need_repo.

``` ruby
@random_repo = Repo.rand.not_in(@most_repo.id, @need_repo.id).first || @most_repo || @need_repo
```

Reply:
good call! this keeps the code logic more similar to the one intended originally. I've already committed the change.